README.md

```diff
@@ -80,7 +85,7 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   ```
   rosrun godel_network irb2400_client.sh [server ip] [robot namespace] true
   ```
-  Note:Robot namespce is the identity of individual robot, can't be the same.
+  Note:Robot namespace is the identity of individual robot, they can't be the same.
```

Comment:
Robot namespace is the identity of individual robot (simulated or real), and service side can run different processes based on different namespaces, but these namespaces can't be the same.